kafka/build.yml

```diff
@@ -1,5 +1,9 @@
 repository: monasca/kafka
 variants:
+  - tag: 0.9.0.1-2.11-1.1.0
+    args:
+      KAFKA_VERSION: "0.9.0.1"
+      SCALA_VERSION: "2.11"
   - tag: 0.9.0.1-2.11-1.0.2
```

Comment:
@kornicameister I think so, yes. It's less of an issue for images that use date+time stamped tags since they'll always end up unique, but we definitely want to avoid introducing breaking changes by overwriting a tag that looks like it should be unique (like semver)